Include upgrading to Qiskit 1.0 on install page (#634)
Qiskit/documentation: docs/start/install.mdx

Whether you will work locally or in a cloud environment, the first step for all users is to install Qiskit. For those wanting to run on a real system, your next step is to choose one of two channels in order to access IBM Quantum systems: IBM Quantum Platform or IBM Cloud.

> <span id="qiskit-upgrade"></span>

Reviewer: ```suggestion
<span id="qiskit-1-upgrade-venv"></span>
```

> ## Upgrade to Qiskit 1.0

Reviewer: ```suggestion
## Upgrade to Qiskit 1.0 requires deleting virtual environment
```

Author: @Eric-Arellano the blog doesn't mention explicitly deleting the current virtual environment. Is that implied when creating a new one? And do we need extra instructions on how to delete the current one?

Reviewer 2: I'm not sure how best to do this, but I think we should make it clear somehow that this info is important for _existing_ users of qiskit. Brand new users can use the install page as normal. Maybe just the header should be changed, or else the first line could tell people to skip this section if they are a new user?

<Admonition type="info" title="You must create a new virtual environment to install Qiskit 1.0">

>   Because Qiskit 1.0 uses a new packaging structure, you cannot upgrade an existing Qiskit 0.x installation to Qiskit 1.0 in-place. Future updates starting with Qiskit 1.0 will allow for in-place upgrades.  

Reviewer: ```suggestion
  Because Qiskit 1.0 uses a new packaging structure, you cannot upgrade an existing Qiskit 0.x installation to Qiskit 1.0 in-place. That is, you must fully delete your virtual environment and completely reinstall it when upgrading.
  
  Once you've upgraded to Qiskit 1.0, you'll be able to upgrade in-place like normal again.
```

Author: Thanks @Eric-Arellano  -- I'm still in the draft stage (as marked) and am just copying and pasting from the blog as a placeholder -- would you mind waiting till it's in review to give comments? :D

Reviewer: Ah sure, my bad!

Author: No problem! I appreciate your close attention to detail. I'll tag you as reviewer when I'm ready 😃

Whether you will work locally or in a cloud environment, the first step for all users is to install Qiskit. For those wanting to run on a real system, your next step is to choose one of two channels in order to access IBM Quantum systems: IBM Quantum Platform or IBM Cloud.

<span id="qiskit-1-upgrade-venv"></span>

> ## Upgrade to Qiskit 1.0

Author: ```suggestion
## For current Qiskit users: Upgrade to Qiskit 1.0
```

> <Admonition type="info" title="You must create a new virtual environment to install Qiskit 1.0">

Author: ```suggestion
<Admonition type="info" title="Current users must create a new virtual environment to install Qiskit 1.0">
```

  Because Qiskit 1.0 uses a new packaging structure, you cannot use `pip install -U qiskit` to upgrade from any Qiskit 0.x version to 1.0. Instead, create a new virtual environment following the following steps. 
  
  Future updates starting with Qiskit 1.0 will allow for in-place upgrades.  
</Admonition>

> To upgrade to Qiskit 1.0, create a new virtual environment. Here is an example, using the standard tool *venv*:

Author: ```suggestion
(If you do not have a Qiskit installation already, skip this section.) To upgrade to Qiskit 1.0, create a new virtual environment. Here is an example, using the standard tool *venv*:
```

> (If you do not have a Qiskit installation already, skip this section.) To upgrade to Qiskit 1.0, create a new virtual environment. Here is an example, using the standard tool *venv*:

Author: ```suggestion
(If you do not have a Qiskit installation already, skip ahead to the [Install and set up](#local) section.) To upgrade to Qiskit 1.0, create a new virtual environment. Here is an example, using the standard tool *venv*:
```

```shell
# Create a new virtual environment (you only do this once)
python -m venv ~/qiskit-1.0-env

# Activate the new virtual environment (you do this every time you start a new session)
source ~/qiskit-1.0-env/bin/activate

# Install Qiskit 1.0 (you only do this once for this environment)
pip install ‘qiskit>=1.0’
```

> Note that you may need to adapt your code to work with Qiskit 1.0, as there will be some breaking changes with respect to 0.x; see RELEASE NOTES OR MORE DETAILS LINK HERE

Reviewer 2: I think the release notes would be the appropriate thing to link here

> Note that you may need to adapt your code to work with Qiskit 1.0, as there will be some breaking changes with respect to 0.x; see RELEASE NOTES OR MORE DETAILS LINK HERE

Author: ```suggestion
Note that you may need to adapt your code to work with Qiskit 1.0, as there will be some breaking changes with respect to 0.x; see the [Qiskit 1.0 migration guide](/api/migration-guides/qiskit-1.0) for details.
```
